sqltest/generic.go

```diff
@@ -12,6 +12,11 @@ const (
 	showTablesStmt  = "SHOW TABLES"
 )
 
+var (
+	// TruncateTimout is the time in seconds for truncation calls. (Default: 5 seconds)
+	TruncateTimout = 5 * time.Second
```

Comment:
should be `TruncateTimeout`

Reply:
`e` button is broken, blame Apple

Reply:
🌚

Reply:
![image](https://user-images.githubusercontent.com/3737347/85748056-b2266d80-b6ff-11ea-9fac-bb0f8c27d875.png)

Sad times

Reply:
Feelsbadman